src/web/package.json

```diff
@@ -38,7 +38,8 @@
     "react-use": "17.3.2",
     "smoothscroll-polyfill": "0.4.4",
     "swr": "1.2.2",
-    "yup": "0.32.11"
+    "yup": "0.32.11",
+    "@testing-library/react": "12.1.4"
   },
   "devDependencies": {
     "@senecacdot/eslint-config-telescope": "1.0.1",
```

Comment:
The ESLint is failing but it's expected due to the `tools/eslint/index.js` changes in this PR
https://github.com/Seneca-CDOT/telescope/blob/4d411c6f58c1bac42fc14264aed9d33cc0feb4fd/tools/eslint/index.js#L14-L16

This `import/resolver` will resolve the 155 errors. To pass ESLint locally, change this line to
```
"@senecacdot/eslint-config-telescope": "workspace:1.0.1",
```

Reply:
Thanks @cindy.
Or you can manually link eslint: `cd <subdir> && pnpm link ./<relativepath-to-eslint>

Reply:
Err you tagged angry-raccoon-python-developer-cindy. Also good to know the pnpm link method. Knowing CLI commands makes things shorter :)

Reply:
lol, those names are always taken

Reply:
Unfortunately, I don't have an SEO-friendly name. Maybe when I'm married